### PR TITLE
docker-clean 2.0.3 (new formula)

### PR DESCRIPTION
**Pull request by killianbrackey**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/docker-clean.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Docker-Clean Script
+# Maintained by Killian Brackey && Sean Kilgarriff at ZZROT Design
+# Contact us at docker@zzrot.com for any issues or fork and pull from
+# the Github repo below.  Happy cleaning!
```
Comment (mikemcquaid) [inline Formula/docker-clean.rb]: Please remove these comments; you can't actually maintain this as you don't have commit access 😉

Comment (Skilgarriff) [inline Formula/docker-clean.rb]: Can we at least add a comment back to the original repo - for issue management? Otherwise we are on it!

Comment (UniqMartin) [inline Formula/docker-clean.rb]: Nobody will see these comments except for Homebrew maintainers and other contributors (like you) who will update this formula in the future. But those people will also see the `homepage` line below. Regular users are much more likely to consult `brew info docker-clean` or use `brew home docker-clean` to get to the right place to report upstream issues.

Comment (mikemcquaid) [inline Formula/docker-clean.rb]: > But those people will also see the homepage line below. Regular users are much more likely to consult brew info docker-clean or use brew home docker-clean to get to the right place to report upstream issues.

This 👍

**Comment by killianbrackey** [comment]
Great, thanks for the input!  I removed the comments and updated a version to include a recent hotfix.

File: Formula/docker-clean.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class DockerClean < Formula
+  desc "Script cleans Docker containers, images, networks, and volumes."
```
Comment (UniqMartin) [inline Formula/docker-clean.rb]: Please tweak the description and omit the full stop at the end. I think the following is better and more concise:

```ruby
desc "Clean Docker containers, images, networks, and volumes"
```

File: Formula/docker-clean.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  homepage "https://github.com/ZZROTDesign/docker-clean"
+  url "https://github.com/ZZROTDesign/docker-clean/archive/v2.0.3.tar.gz"
+  sha256 "a2e1923fe2862ca2487e39000c68cba900711b768a8920dd1b1275a2fa0e6789"
+
+  def install
```
Comment (UniqMartin) [inline Formula/docker-clean.rb]: This formula doesn't need any compilation or other similar things, thus it doesn't make sense to provide a bottle (binary package) for it. Please add `bottle :unneeded` followed by an empty line above this line here.

**Comment by killianbrackey** [comment]
Thanks for the help and quick responses.  Let me know if there is anything else!

**Comment by UniqMartin** [comment]
Merged. Thank you for this contribution to Homebrew, @killianbrackey! :tada:

For future reference: If you can, please update your PR by amending your commit and then pushing it. We prefer to have one commit per formula. Also, you can avoid issues like the checksum mismatch or trailing whitespace by running `brew install` and `brew audit` (as suggested in the checklist in the top post) before re-submission of your changes. (It's also often possible to configure a text editor to automatically remove trailing whitespace on save, if haven't considered this yet.)

**Pull request merged**